vividus-allure-adaptor/src/test/java/org/vividus/report/allure/adapter/AllureReportGeneratorTests.java

```diff
@@ -216,6 +218,7 @@ private LoggingEvent buildReportGeneratedLogEvent(Path reportDirectory)
     private void testEnd(File reportDirectory) throws IOException
     {
         Path historyDirectory = tempDir.resolve("history");
+        List<Plugin> plugins = List.of();
```

Comment:
it's better to add some plugin mock to this list

Reply:
The problem here is that tests for AllureReportGenerator are not atomic and this list will be used inside external allure logic. So with mocking of list we will need additionally mock all interactions with this mock.